README.md

```diff
@@ -106,55 +108,31 @@ Class parameters:
 * **redis_port**: port to use for Redis (6379)
 * **secret_key**: string used to hash cookies (fqdn_rand_string(40))
 * **smtp_host**: name or IP of SMTP server (localhost)
-* **ssl_* **: Apache SSL controls
-* **team**: name of the default team to create and use for new projects
-* **url**: source URL form which to install Sentry (false, use PyPI)
+* **ssl_* **: Apache SSL controls.  The `ssl_cert` and `ssl_key` parameters reference files on the Sentry server.  You are responsible for getting these files onto the server yourself.
```

Comment:
Maybe change the wording here to "Your sentry profile class or other higher level class should manage the installation of these file resources.".